src/services/ai.ts

```
@@ -0,0 +1,641 @@
+            text: prompt
+          }]
+        }],
+        generationConfig: {
+          temperature: 0.7,
+          maxOutputTokens: 4000,
+        },
+      });
+
+      const content = response.data.candidates[0].content.parts[0].text;
+      return this.parseResponse(content, context.questionCount);
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)